apps/files/lib/activityhelper.php

```
@@ -47,7 +47,11 @@ public function __construct(ITagManager $tagManager) {
 	 */
 	public function getFavoriteFilePaths($user) {
 		$tags = $this->tagManager->load('files', [], false, $user);
-		$favorites = $tags->getFavorites();
+		try {
+			$favorites = $tags->getFavorites();
+		} catch (\Exception $e) {
+			throw new \RuntimeException('No favorites', 1);
```

Can we please not simply swallow any exception. Either add it as the third parameter (previous exception https://php.net/manual/de/exception.construct.php ) or append the message or log the previous message, but please don't drop it silently. :)